### PR TITLE
Extend readme2024

### PR DESCRIPTION
**Pull request by alxvth**
Mention (most) features and add a screenshot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
+- Overview of image information for all image files in a given directory:
+  - Size, number of points (pixels), channels and more
+  - Filter images by name
+- Edit meta data like data set and dimension names
+- Select which dimensions are to be loaded
+- Multiple image formats: PNG, JPEG, BPM and (single page) TIF/TIFF
+  - RGB images
+  - "High-dimensional" images, i.e. more than three values per pixel. All grayscale (single channel) image files in a folder are interpreted as part of one single image, e.g. the files in `./data/images/stack/sun/`
+- Data layout interpretation:
+  - Image sequence (default): Each grayscale image in a folder is interpreted as an image channel, i.e., given 100 images of 28x28 pixels this loader creates a dataset of 28x28=784 data points with 100 dimensions (values per pixel). The image dimensions is 28x28 pixel.
```
Comment (ThomasKroes) [inline README.md:22]: The image stack layout is the default, I would describe that one first:
- Image stack (default): In this data layout, same-sized images are layered on top of each other. Pixels with the same coordinate in each layer together form a data point, where each element represents a dimension. Given 100 images of 28x28 pixels, this loader creates a dataset of 28x28=784 data points with 100 dimensions (values per pixel). The image dimensions is 28x28 pixels.
- Image sequence: In this case, each image of a same-size sequence of images is considered a data point and each pixel is considered a dimension. Given 100 images of 28x28 pixels, this loader creates a dataset of 100 data points with 28x28=784 dimensions.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
+  - Image stack: Each pixel of all grayscale images in a folder are interpreted as an image channel, i.e., given 100 images of 28x28 pixels this loader creates a dataset of 100 data points with 28x28=784 dimensions. The image dimensions remain 28x28 pixel.
+- Apply data conversion:
+  - If a transformation plugin is installed, like the [PointDataConversionPlugin](https://github.com/ManiVaultStudio/PointDataConversionPlugin), you can apply the transformation that it provided upon loading, e.g. `log2` conversions.
+- Subsample images:
+  - Resize an image with linear, bilinear, B-spline, bicubic, Catmull-Rom or Lanczos interpolation filter
+  - Create a image pyramid of user-defined size by downsampling the image on every level 
+
```
Comment (ThomasKroes) [inline README.md:29]: ### Multi-page TIFF
This loader can import layered TIFF images. To also import the dimension name for each image, one can use the user interface as shown below to select the image `TAG` to use for the dimension name:
image

**Review by ThomasKroes — CHANGES_REQUESTED** [review]
I like the readme overall; I have a few comments/suggestions

File: README.md (hunk 1 above)
Comment (alxvth) [reply README.md]: Ah, I got them mixed up! I fixed that and expanded the explanation

**Review by ThomasKroes — APPROVED** [review]

**Pull request merged**